ui/components/component-library/button-icon/button-icon.types.ts

```diff
@@ -1,10 +1,10 @@
 import type { BoxProps } from '../../ui/box/box.d';
-import { IconName, IconSize } from '../icon';
+import { IconName, IconProps } from '../icon';
 import { Color, IconColor } from '../../../helpers/constants/design-system';
 
 export enum ButtonIconSize {
-  Sm = IconSize.Sm,
-  Lg = IconSize.Lg,
+  Sm = 'sm',
+  Lg = 'lg',
```

Comment:
I've spoken with Brad and Elliot and unfortunately we don't have a good way to map one enum value to another so we will just have to be vigilant and use the same strings across the board for our size enums